app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError("The visitor doesn't have a vaccine!")
```

Comment:
what if vaccine is an empty dict? The error will not be raised but the line below will raise a key error